lib/src/animation_panel.dart

```diff
@@ -36,13 +38,11 @@ class AnimationPanel<T> extends StatelessWidget {
           : duration ?? const Duration(milliseconds: 200),
       curve: curve!,
       width: width,
-      child: PhysicalModel(
-        elevation: elevation!,
-        clipBehavior: Clip.antiAlias,
-        color: color!,
-        borderRadius: BorderRadius.circular(radius!),
-        child: Container(height: height),
+      decoration: decoration?.copyWith(
```

Comment:
if you do not pass "thumbDecoration" then "AnimationPanel" will not be displayed, this behavior is not suitable, then you will have to make the parameter "thumbDecoration" mandatory or add a default display of the panel

Reply:
I suggest making it so that there is support for the current logic, and not making "thumbDecoration" mandatory

decoration: decoration?.copyWith(
            borderRadius:
                decoration?.borderRadius ?? BorderRadius.circular(radius),
            color: decoration?.color ?? color,
          ) ??
          BoxDecoration(
            borderRadius:
                decoration?.borderRadius ?? BorderRadius.circular(radius),
            color: decoration?.color ?? color,
          ),